md/datetimes-management-tutorial.md

```diff
@@ -4,88 +4,89 @@
 
-1. In the process **execution** tab > **Contract** section, define the contract inputs using the **Add from data...** button
-1. Select the business variable *employee*
-1. Follow the steps to generate a contract with a COMPLEX input *employeeInput* containing a **DATE ONLY** attribute (keep the option to generate the variable initialization script automatically)
+1. In the process **Execution** tab > **Contract** section, define the contract inputs using the **Add from data...** button
+2. Select the business variable *employee*
+3. Follow the steps to generate a contract with a COMPLEX input *employeeInput* containing a *birthdate* attribute of type **DATE ONLY** (keep the option to generate the variable initialization script automatically)
 
 ### Generate the process instantiation form
 
 1. In the process **Execution** tab, **Form** section, generate the form by clicking on the **pencil** icon
-1. Since you have a **DATE ONLY** attribute in the contract inputs, a **Date picker** widget is automatically added to the form  
+2. Since you have a **DATE ONLY** attribute in the contract inputs, a **Date picker** widget is automatically added to the form  
 The form is already configured correctly.
 
-### Define the task contract and operations to update the date
+### Define the contract at task level, as well as operations to update the business variable
 
 1. Rename the first task to *Update birth date*
-1. In the task execution tab > **Contract** section, define the contract inputs using the **Add from data...** button
-1. Select the business variable *employee*
-1. Follow the steps to generate a contract with a COMPLEX input *employeeInput* containing a *birthdate* attribute of type **DATE ONLY** (keep the option to generate the operations automatically)
+2. In the task **Execution** tab > **Contract** section, define the contract inputs using the **Add from data...** button
+3. Select the business variable *employee*
+4. Follow the steps to generate a contract with a COMPLEX input *employeeInput* containing a *birthdate* attribute of type **DATE ONLY** (keep the option to generate the operations automatically)
 
 ### Generate the 'Update' task form
 
 1. In the task **Execution** tab, **Form** section, generate the form by clicking on the **pencil** icon
-1. Since you have a **DATE ONLY** attribute in the contract inputs, a **Date picker** widget is automatically added to the form  
-Now in the UI Designer, define a variable to retrieve the the *Employee* currently stored using the context:  
-1. In the **Variables** panel at the bottom of the editor, click on **Create a new variable**
-1. Enter the name *employee* for the new variable
-1. select the **type** *external API*
-1. Set the **API URL** of the variable to `../{{context.employee_ref.link}}`  
+2. Since you have a **DATE ONLY** attribute in the contract inputs, a **Date picker** widget is automatically added to the form  
```

Comment:
anyway, it will still work if you write:
`1.
3.`
So it does not really matter